snakemake/executors.py

```diff
@@ -897,12 +897,20 @@ def job_status(job):
 
             def job_status(job):
                 if os.path.exists(active_job.jobfinished):
-                    os.remove(active_job.jobfinished)
-                    os.remove(active_job.jobscript)
+                    if self.workflow.cleanup_wrappers:
```

Comment:
From your issue, this is actually not what you want to achieve I think. The jobscript is a shell script that is meant for submission to classical cluster systems.

I think what you meant to modify is the behavior here: https://github.com/snakemake/snakemake/blob/master/snakemake/script.py#L567
In other words, if your flag is false, that line shall not be executed.

And then, the flag should be called `--skip-cleanup-scripts`.

Reply:
Ah, there it is! Thank you!